stdlib/open-uri/0/open-uri.rbs

```diff
@@ -0,0 +1,33 @@
+module URI
+  def self.open: (String name, ?String mode, ?Integer perm, ?untyped options) -> (StringIO | Tempfile)
```

Comment:
This return type is a bit incorrect. `URI.open` returns a StringIO (or a Tempfile) object extending by `OpenURI::Meta`. But it's difficult to declare it to me...

Reply:
@soutaro taught me the way to represent types for mix-ins, and I updated this definition now.
Resolved.

Thanks as always!